src/components/Carousel/Carousel.js

```diff
@@ -111,18 +132,41 @@ class Carousel extends Component {
     });
   }
 
+  renderChildren() {
+    const { children, items } = this.props;
+    const { widthOfSlide } = this.state;
+    const style = { maxWidth: `${widthOfSlide}px` };
+    const listOfChildren = [...children];
```

Comment:
Children is a list of objects, this thing will create a shallow copy. Links to objects would keep same. Make sure objects wouldn't be modified

Reply:
done)